src/machine/behavior/is/mod.rs

```diff
@@ -288,3 +291,262 @@ pub const SEMANTICS: [Semantic; 256] = [
     other::get,
     interrupt::trip,
 ];
+
+const INSTR_NAMES: [&'static str; 256] = [
+    "interrupt::trap",
```

Comment:
Donald Knuth names the instructions `TRAP`, `FCMP`, ..
Maybe we should not introduce our own names here!